app/javascript/components/subject/MonitoringStatus.js

```diff
@@ -193,8 +212,20 @@ class MonitoringStatus extends React.Component {
     this.setState({
       message: 'jurisdiction from "' + this.props.jurisdictionPaths[this.state.originalJurisdictionId] + '" to "' + this.state.jurisdictionPath + '".',
       showJurisdictionModal: !current,
-      jurisdiction: current ? this.state.jurisdictionPath : this.props.jurisdictionPaths[this.state.originalJurisdictionId], // Reset select jurisdiction if cancel
+      jurisdictionPath: current ? this.props.jurisdictionPaths[this.state.originalJurisdictionId] : this.state.jurisdictionPath,
```

Comment:
the older version of this line was incorrect, this is supposed to reset the jurisdiction path text field to the original jurisdiction path if the user clicks cancel